apps/site/lib/site_web/templates/customer_support/_form.html.eex

```diff
@@ -64,9 +64,10 @@
         </div>
       </div>
       <div class="plan-time-select">
-        <%= label f, "Vehicle", [id: "vehicleLabel", for: "support_vehicle", class: "form-control-label"] %>
-        <div>
-          <%= text_input(f, :vehicle, placeholder: placeholder_text("vehicle"), class: "plan-date-input") %>
+        <%= support_error_tag @errors, "vehicle" %>
+        <div class="form-group <%= class_for_error("vehicle", @errors, "has-danger", "has-success") %>">
+          <%= label f, "Vehicle", [id: "vehicleLabel", for: "vehicle", class: "form-control-label"] %>
+          <%= text_input(f, :vehicle, id: "vehicle", placeholder: placeholder_text("vehicle"), class: "form-control plan-date-input", value: assigns[:vehicle] ) %>
```

Comment:
Is it worth using the HTML5 regex validator here as well? It's not supported everywhere, but it's pretty wide: https://caniuse.com/input-pattern

Reply:
I thought about using this... I think its main benefit would be a more informative message could be offered by the browser, but the way our front-end validation is currently set up the user wouldn't see that message anyway. Which is not super great, but for that reason I chose to not use the input pattern attribute here (or elsewhere in the form).